translations/manage/participation/th.json

```diff
@@ -0,0 +1,25 @@
+{
+  "title": "การเข้าร่วมระบบ MySK",
+  "desc": "ตัวชี้วัดการเข้าร่วมต่าง ๆ ของนักเรียนและครูในการใช้ระบบ MySK",
+  "awareness": {
+    "alt": "แผนภูมิจำนวนผู้ใช้เข้าร่วม",
+    "overline": "การเข้าถึง",
+    "title": "{{percentage, number(style: percent;)}} ของผู้ใช้ทั้งหมดเข้าสู่ระบบอย่างน้อยหนึ่งครั้ง",
```

Comment:
This results in a translation error, but the English translation is formatted properly. I'll commit you a fix after this comment!

<img width="770" alt="Screenshot 2567-11-25 at 09 53 53" src="https://github.com/user-attachments/assets/5b6834fa-e8ca-44a9-9698-4391783616a4">

<sub>p.s. Next time also check for both Thai and English! I think that's where most of our translation problems came from.</sub>